components/library/tntnet/Makefile

```diff
@@ -37,6 +37,11 @@ include $(WS_TOP)/make-rules/ips.mk
 COMPONENT_PRE_CONFIGURE_ACTION = \
         ($(CLONEY) $(SOURCE_DIR) $(@D))
 
+COMPONENT_POST_INSTALL_ACTION += \
+        (cd $(PROTO_DIR)/etc/tntnet && \
+         $(GSED) -i 's|>www-data<|>webservd<|g' tntnet.xml && \
+         $(GSED) -i 's|/$(MACH64)/|/|g' tntnet.xml )
+
```

Comment:
at least you can eliminate last gsed by rule "$(INSTALL_32):         $(INSTALL_64)"